tests/test_common.sh.in

```diff
@@ -314,4 +314,14 @@ xsed() {
 	esac
 }
 
+soft_die() {
```

Comment:
Something tells me that after calling this function nothing would die, neither in a hard and painful way nor peacefully. Can we call it `logerr` or something along the lines of its duty?

Reply:
The name is tricky, it is both about logging to stderr and about returning non-zero code. What about `log_failure`?

Reply:
Yep, that will do, I guess. But maybe, to be even clearer for people who would read the test code, something like
```bash
check_something "blah blah" || log_and_fail "Print error here and return 1"
```
Not sure how useful it is, though. We can always just use `die` in this case.